CLOUD-2178: Flexibility, Describe how to copy data from one instance … (#1522)
akeneo/pim-docs: cloud_edition/flexibility_mode/docs/environments_access.rst

.. _`specific tasks`: ./partners.html
.. _`GitHub Help Center`:  https://help.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent
.. _`Akeneo Help Center`:  https://help.akeneo.com/portal/articles/access-akeneo-flexibility.html?utm_source=akeneo-docs&utm_campaign=flexibility_partner_starterkit

Copy data from one instance to another
**************************************

**Scenario:**

    User wants to transfer a SQL dump file from production instance their staging instance:
    
.. note::

    **User** connects with SSH and forwards their local ssh-agent towards
        > **project.akeneo.cloud.com**, and runs scp command towards
        
        > **project-staging.akeneo.cloud.com**

    👨‍💼 💻   ──────> 🔑  ──────> 🖥  ────── 🔑 ──────> 📂 ──────> 🖥

**Prerequisites:**

> - Get ssh key access to both intances for akeneo user.

Reviewer: ```suggestion
- Get SSH key access to both intances for akeneo user.
```

- Get network access to instances.

**Usage:**

.. code-block:: bash

    eval `ssh-agent`
    ssh-add ~/.ssh/id_rsa
    ssh -tA akeneo@project-staging.cloud.akeneo.com
    
    scp dump.sql akeneo@project:dump.sql

.. warning::

>     For the scp step, please note that no domain is specified.

Reviewer: ```suggestion
    On the SCP command, please note that no domain is specified.
```

>     Set the short hostname of instance. (connect to the target server and run command : `hostname`)

Reviewer: ```suggestion
    Use the short host name of instance. Connect to the target server and run `hostname` to get this value.
```

.. _`specific tasks`: ./partners.html
.. _`GitHub Help Center`:  https://help.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent
.. _`Akeneo Help Center`:  https://help.akeneo.com/portal/articles/access-akeneo-flexibility.html?utm_source=akeneo-docs&utm_campaign=flexibility_partner_starterkit

> Copy data from one instance to other

Reviewer: ```suggestion
Copy data from one instance to another
```

************************************

**Scenario:**

>     `User` want to transfer a SQL dump file from prod instance toward staging instance:

Reviewer 2: Backquotes are not interpreted in this block

Author: removed

>     `User` want to transfer a SQL dump file from prod instance toward staging instance:

Reviewer: ```suggestion
    `User` wants to transfer a SQL dump file from production instance their staging instance:
```

    
.. note::

>     **`User`** ssh connect with ssh-agent forwarding towards

Reviewer: ```suggestion
    **`User`** connects with SSH and forwards their local ssh-agent towards
```

Author: done

>         > **`project.akeneo.cloud.com`**, and run scp command towards

Reviewer: ```suggestion
        > **`project.akeneo.cloud.com`**, and runs scp command towards
```

Author: done

        
            > **`project-staging.akeneo.cloud.com`**

    👨‍💼 💻   ──────→ 🔑  ──────→ 🖥  ──────→ 📂 ──────→ 🖥
        
      

**Prerequisites:**

> - Get ssh key access to intance for akeneo user.

Reviewer 2: ```suggestion
- Get ssh key access to both intances for akeneo user.
```

- Get network access to instances.

**Usage:**

.. code-block:: bash

    eval `ssh-agent`
    ssh-add ~/.ssh/id_rsa
    ssh -tA akeneo@project-staging.cloud.akeneo.com
    
    scp dump.sql akeneo@project:dump.sql

.. warning::

>     For the scp step, note that no domain is mentioned.

Reviewer: ```suggestion
    For the scp step, please note that no domain is specified.
```
